packages/compass-components/src/components/leafygreen.tsx

```diff
@@ -102,6 +103,10 @@ export {
   Icon,
   IconButton,
   Language,
+  // We export the GuideCue as LGGuideCue as we have our own wrapped
+  // GuideCue implementation in Compass. LGGuideCue is only to be
+  // used in situations where Compass' GuideCue is not intended for use.
+  LGGuideCue,
```

Comment:
This will be eventually used for aggregation right? Maybe worth putting this component to compass-components as some sort of FeedbackTooltip instead of exposing lg guide cue like that?

Reply:
No strong preference on this! Just thinking how we can keep GuideCue internal 🤔

Reply:
Good thinking, updated. It does still feel like GuideCue is maybe not the right component for this from LeafyGreen, I think it's better than custom popovers for now, we can revisit down the line I reckon.
I went with FeedbackPopover instead of FeedbackTooltip if that's alright with you, I can switch it back if you prefer.

Reply:
No, no preference, both make sense, I guess we could use an InteractivePopover, but it doesn't have a tooltip pointer thingy, right?

Reply:
No tooltip pointer thing yup. Let's see if we end up doing more of this sort of thing in which case we can add something to the interactive popover for that and use it here.